public/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>fileFU  |  HOME</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
You should add a link to the global.js file here as well. Makes the inline code below body tag extraneous. 

   ```<script src="./javascript/global.js"></script>```